### DISCUSSION TITLE
iOS sync

### DISCUSSION THREAD

**Discussion by lukethehawk** [advplyr/audiobookshelf-app#639] [General]
On iOS 16.3.1
App v. 0.9.63 (20)

I notice that the synchronization between app and server in this direction (app->server) gave me some problems, it happened that the app did not synchronize with the server, so if I logged into the web page I saw that the listening time was lower .
If I logged out and logged back into the app it would re-sync with the server and put me further back, as if the app's progress hadn't been recorded.

Is this a known bug?

thanks

**Comment by advplyr** [comment]
Issues with progress sync on iOS has been mentioned a few times but no one has been able to provide reproducible steps yet. Once we have reproducible steps it can get fixed.

**Reply by lukethehawk** [reply]
Ok thanks

**Comment by hobesman** [comment]
Are you using a downloaded book? Or streaming from your server?

**Reply by lukethehawk** [reply]
Streaming from server

**Comment by Eragos** [comment]
@advplyr 
I'm noticing some sync issues last few days. I thought in the last days a “pull to refresh” function could prevent this behavior and fetch the current status from the server again 💡

By the way, it always happens when I play from another device (iPhone, iPad, web, e.g.) and switch back. It seems as if the app does not pay attention to a different playback status of the serverside. I currently always hard close the app (task).

Best, Michael

> PS: _I started - coming from pocket casts and I use mostly Podcasts - a few weeks ago and moved now completely over to Audiobookshelf. I noticed the TestFlight was full, but I compiled it for myself. My Developer account was indeed helpful and now I do not block a TestFlight slot 😉_
